src/app/modules/products/products.module.ts

```diff
@@ -1,20 +1,23 @@
-import { NgModule } from '@angular/core';
-import { CommonModule } from '@angular/common';
+import { NgModule } from "@angular/core";
+import { CommonModule } from "@angular/common";
 import { SharedModule } from "@shared/shared.module";
 
-import { ProductsRoutingModule } from './products-routing.module';
+import { NgxSkeletonLoaderModule } from "ngx-skeleton-loader";
```

Comment:
these loaders may be used for other modules as well. Therefore, they should be imported in the shared module